client/actions/projects.js

```diff
@@ -370,7 +373,7 @@ export function fetchQuestionVersions(key) {
 
 const debouncedSyncProject = debounce((...args) => {
   return syncProject(...args);
-}, 1000, { maxWait: 5000, leading: true });
+}, 2000, { maxWait: 20000, leading: false });
```

Comment:
This should help reduce a bit of API load too since it throttles down the rate at which changes sync if the application is changing continually.